### ISSUE TITLE
Missing python3-config error is not terribly helpful

### ISSUE THREAD

**Issue by Ketsuban** [chergert/gnome-builder#38]
`Failed to locate python3-config` is not particularly actionable; I had to google the error and serendipitously found a forum thread where someone worked out the solution was to install the package `python3-dev`. Another error I got while playing development package whack-a-mole told me what package to install; could this error do likewise?

I could probably have done this myself and submitted a pull request, but had a look at the `configure.ac` file and wasn't confident enough in what I was doing; I'd rather someone with more `autotools` experience handle it.

**Comment by TingPing** [comment]
I think that is about as good an error as you can get; All configure wants is the binary `python3-config`, it does not have knowledge of your distro/package manager/where to download it/etc.

A pro tip for you though is `apt-file search python3-config` which will say what package provides it.

**Comment by Ketsuban** [comment]
True, it can't say with certainty, but the other error I mentioned wasn't actually right either—I think it gave me a package name for Fedora, which happened to be close enough to the Ubuntu equivalent for me to get it. (Actually, in mid-typing I remembered what it was—`llvm-devel` and `clang-devel`, which turned into `llvm-dev` and `libclang-dev`.)

**Comment by TingPing** [comment]
Exactly why putting package names in error messages does more harm than good :P. Usually once you get familiar with a distro though you do learn about how they split (or don't) packages into -dev, -utils, -cli (debian takes it to the extreme..).
